build_cmake/FrameworkOpenAl/CMakeLists.txt

```diff
@@ -19,4 +19,5 @@ set(SRC_FILES
 )
 
 add_library(Framework_OpenAl ${SRC_FILES})
+target_link_libraries(Framework_OpenAl PUBLIC ${OPENAL_LIBRARY})
```

Comment:
Could that be moved in a separate commit? It doesn't really have anything to do with making sure `string_cast_win1252.cpp` compiles.

Reply:
moved to the previous commit